apis/task.py

```diff
@@ -48,13 +48,20 @@ class TpuTask(BaseTask):
     task_test_config: Test configs to run on this TPU.
     task_gcp_config: Runtime TPU creation parameters.
     task_metric_config: Metric configs to process metrics.
+    custom_tpu_name: A custom TPU name.
+    tpu_name_with_suffix: The flag to define if add auto-generated suffix.
+    all_workers: The flag to define if run commands on all workers or worker 0
+      only.
   """
 
   # TODO(wcromar): make these attributes less verbose
   task_test_config: test_config.TestConfig[test_config.Tpu]
   task_gcp_config: gcp_config.GCPConfig
   tpu_create_timeout: datetime.timedelta = datetime.timedelta(minutes=60)
   task_metric_config: Optional[metric_config.MetricConfig] = None
+  custom_tpu_name: str = ''
```

Comment:
nit: use `Optional[str]` with `None` as the default value instead

Reply:
Sounds good. Will update.

Reply:
Sounds good.